frontend/src/bundles/video-editor/components/timeline/subcomponents/timeaxis.tsx

```diff
@@ -0,0 +1,118 @@
+import { minutesToMilliseconds } from 'date-fns';
```

Comment:
why we need this lib?

Reply:
waiting for answer))

Reply:
Im sorry. We dont need it, it just multiplies by 60000. Removed from frontend dependencies and moved to devDependencies as it is used to create the mock data.

Reply:
With such simple logic, you can create just a helper. To avoid installing new dep